README.md

```diff
@@ -59,32 +59,35 @@ It's _not_ a repository for all shared components. Don't try to add components t
 
 For simple projects, you should be able to use [`npm link`](https://docs.npmjs.com/cli/link.html) or [`yarn link`](https://yarnpkg.com/lang/en/docs/cli/link/) to temporarily and invisibly load your local copy of RoverUI as a dependency in another project. These tools both have caveats, though. They may fail on peer dependencies, and their symlinking strategy can introduce bugs in monorepos with multiple workspaces.
 
-**Experimental**
+#### Testing in monorepos with `yalc` (experimental)
 
 You can use [yalc](https://github.com/whitecolor/yalc) to test a local project. It copies files instead of symlinking them.
 
 1. In the RoverUI project directory, use `yalc publish --force`. Using `--force` is important because it skips the publish scripts, so it won't try to connect to npm.
-2. In the target project directory, use `yalc add @cision/rover-ui --no-pure`. Using `--no-pure-` seems to be important if you're working in a monorepo with multiple workspaces, but _it will edit your consuming project's `package.json` file and add a `.yalc` folder and `yalc.lock`. **Don't forget to revert those changes when you're finished.**_
+2. In the target project directory, use `yalc add @cision/rover-ui --no-pure`. Using `--no-pure-` is required if you're working in a monorepo with multiple workspaces, but _it will edit your consuming project's `package.json` file and add a `.yalc` folder and `yalc.lock`. **Don't forget to revert those changes when you're finished.**_
 3. You should be able to import RoverUI components in your target project normally now.
 
+Please tread carefully, and add any issues or suggestions on the [GitHub issue tracker](https://github.com/cision/rover-ui/issues).
+
 ## Publishing a new version
 
 To publish the npm package, you'll need a free account on npmjs.com, and you'll need to be added to the list of maintainers by one of the current RoverUI maintainers.
 
-1. Bump the `package.json`'s version _using [semver](https://semver.org/)_. We're currently in alpha, and probably will be for a while. Versions are cheap.
-2. If all tests look good (`yarn test`), submit a pull-request against the master branch through Github. We're using the master branch for now, but by the time we're out of alpha, we'll have a dev branch.
+1. In `package.json`, bump the `version` and `publishConfig.tag` properties _using [semver](https://semver.org/)_. `publishConfig.tag` should start with a "v", but otherwise be identical to `version`.
+2. If all tests look good (`yarn test`), submit a pull-request against the master branch through GitHub.
```

Comment:
I'd still like to see us have a more refined branching strategy, but I don't see any value in a `dev` branch yet since we build and deploy on every approved change.

We could start making a branch-per-version soon, if we start having enough changes per week to merit it.